ruby.c

```diff
@@ -90,6 +90,16 @@ enum feature_flag_bits {
 
 #define DEBUG_BIT(bit) (1U << feature_debug_##bit)
 
+#define HELP_BIT(bit) (1U << help_##bit)
+#define DEFINE_HELP(bit) help_##bit
+#define EACH_HELPS(X, SEP) \
```

Comment:
Hmm, each <plural> doesn't sound right as English.  Should it be fixed along with `EACH_DUMPS`?

Reply:
I wonder which `EACH_HELP` or `HELPS_EACH` is better.
`HELPS_EACH` would be good as it looks like `helps.each {}`?

@nobu what do you think? `EACH_HELP` or `HELPS_EACH` or just keep?